wordpress-mobile/WordPress-iOS
# Site Creation Domain Suggestions: handle when there are no suggestions

Fixes #8429 

To test: 
- Enter a site title that results in no domain suggestions, either on the Site Details page or in the Domain search bar. 
- Verify the view below is displayed.

NOTE: Japanese text returns no suggestions.

Initial search, no suggestions from site title:
![initial_search](https://user-images.githubusercontent.com/1816888/36324578-1ef56f72-1312-11e8-9752-ba74122c5f84.png)

No suggestions from a search via search bar:
![secondary_search](https://user-images.githubusercontent.com/1816888/36324581-21b16c34-1312-11e8-8337-ae372bc45fd1.png)
